packages/timeline/docs/zh-tw/使用說明.md

```diff
@@ -0,0 +1,64 @@
+# @twreporter/timeline 使用說明
+
+這是非營利獨立媒體[《報導者》](https://www.twreporter.org)製作的「大事記產生器」，可以幫你把時間序列資料轉換成網頁版大事記！
+
+## 範例
+
+[範例試算表](https://docs.google.com/spreadsheets/d/1f76OLdfZe3kyNOKiPthWNJWVGmY3bkm5KtxB4NYp9uU/)
+
+![timeline-spreadsheet-screenshot](../assets/spreadsheet.png)
+
+[範例試算表轉換結果](../assets/example.html)
+
+![timeline-present-example](../assets/timeline-present-example.png)
+
+## 使用說明
+
+### 1. 建立你的大事記資料試算表
+
+- [點此建立範例試算表副本](https://docs.google.com/spreadsheets/d/1f76OLdfZe3kyNOKiPthWNJWVGmY3bkm5KtxB4NYp9uU/copy)
+- 按照試算表上的格式說明，填入你的資料。
+
+### 2. 設定共用
+
+- 注意在修改資料時，不要改到資料表的欄位順序，和有底色的保護儲存格。
+- 在`[共用]`設定中，將你的資料表設為「公開」、「知道連結的人均可檢視」、或「知道連結的人均可評論」： ![共用對話框截圖](../assets/share-box.png)
+- 若你想維持資料表只有特定授權使用者可以看到，也可以只授權報導者機器人帳號（timeline@cloud-functions-268910.iam.gserviceaccount.com）讀取你的資料表： ![共用對話框截圖（private）](../assets/share-box-private.png)
+
+### 3. 將你的資料表連結貼到轉換器，取得 embedded code
+
+[轉換器]()
+
+### 4. 將 embedded code 貼到你的網頁
+
+將複製的 embedded code 貼到 `<body>` 和 `</body>` 之間：[原始碼範例](../assets/example.html)
+
+## 版面說明
+
+![timeline example](../assets/timeline-levels.jpg)
+
+### elements
+
+element（元件）是組成一張大事記的最小單位，目前有三種 element：
+
+- `group-flag`: group section 開頭的標題元件
+- `unit-flag`: unit section 開頭的標題元件
+- `record`: 最基本的內容項目，可以有文字和圖片
+
+### sections
+
+一個大事記可以分為多個 sections，目前版面有兩個不同階層的 section：
+
+- `unit`: 最小的 section，由一個 unit-flag 和一個或多個 record 組成
+- `group`: 由一個 group-flag 和一個或多個 unit 組成的 section
```

Comment:
That would be wonderful if we add more examples here to explain `elements` and `sections`.